Networking/Networking/Mapper/StripeAccountMapper.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+/// Mapper: Stripe Account
+///
+struct StripeAccountMapper: Mapper {
+
+    /// (Attempts) to convert a dictionary into an account.
+    ///
+    func map(response: Data) throws -> StripeAccount {
+        let decoder = JSONDecoder()
+
+        /// Needed for currentDeadline, which is given as a UNIX timestamp.
+        /// Unfortunately other properties use other formats for dates, but we
+        /// can cross that bridge when we need those decoded.
+        decoder.dateDecodingStrategy = .secondsSince1970
+
+        /// Detect the exceptional case where we got a response of data:[] indicating
+        /// that the plugin is active but the merchant has not on-boarded (and therefore
+        /// has no account.)
+        if let _ = try? decoder.decode(StripeNullAccountEnvelope.self, from: response) {
+            return StripeAccount.noAccount
```

Comment:
Just curious, any reasons we still want to return a `StripeAccount` instance while there is no account yet?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5799/commits/aba8325b9081168679307c777d8fddafef26e03f